### ISSUE TITLE
Question: Cloud-to-Vehicle Communication Schemas

### ISSUE THREAD

**Issue by zhakhalov** [openmobilityfoundation/mobility-data-specification#349]
I'm inspecting the specification and found **post_vehicle_telemetry** and **post_vehicle_event**. 
Both seem to define the structure of the message sent from vehicle to the application.

Are there schemas for cloud-to-vehicle communication exists or is planned?

**Comment by karcass** [comment]
These are not vehicle-to-cloud _per se_.  Usually these messages flow from the vehicle to the provider, and the provider forwards them.  Some are not from the vehicle, but are generated by the provider mobile app used by a customer, or an internal admin app that the provider uses (e.g. to generate `provider_pick_up` events).

But to answer your question, there are no current agency-to-vehicle endpoints or schemas currently defined.  The community has some ideas for a few specific examples, such as the ability for an agency to send an `unlock` message to a vehicle in order to relocate it.  Other messages from an agency to providers such as `agency_pick_up` ("Hey we impounded this scooter we found in the river") are also up for discussion and eventual implementation.
